Modules/_cursesmodule.c

```diff
@@ -3977,10 +3979,11 @@ _curses_qiflush_impl(PyObject *module, int flag)
 }
 
 /* Internal helper used for updating curses.LINES, curses.COLS, _curses.LINES
- * and _curses.COLS */
+ * and _curses.COLS. */
```

Comment:
Put `Returns 1 on success and 0 on failure.` here.

Reply:
Would it actuall ybe better that I move the "Internal helper" comment inside the `if defined`? That way, we wouldn't have an orphan comment on systems that don't support the feature.